misc/python/materialize/checks/cluster.py

```diff
@@ -18,6 +18,10 @@ def manipulate(self) -> List[Testdrive]:
         return [
             Testdrive(dedent(s))
             for s in [
+                """
+                $[version>=5900] postgres-execute connection=postgres://mz_system@materialized:6877/materialize
+                GRANT ALL PRIVILEGES ON SYSTEM TO materialize
+                """,
```

Comment:
@def- I can't figure out why this is needed. `materialize` should be set as the bootstrap role which should be granted all system privileges by default.

Reply:
Ah I think I know. The role is created in a version without system privileges, then materialize is upgraded to a version with system privileges and this test is run. That's why the failures are only happening in upgrade tests.

Reply:
If you have a better solution then let me know, otherwise this will probably work.

Reply:
This seems fine to me, sorry for the trouble.